libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -1026,7 +1026,7 @@ int HTTPClient::handleHeaderResponse()
 
                 for(size_t i = 0; i < _headerKeysCount; i++) {
                     if(_currentHeaders[i].key.equalsIgnoreCase(headerName)) {
-                        _currentHeaders[i].value = headerValue;
+                        _currentHeaders[i].value += headerValue + ";";
```

Comment:
Can we follow the HTTP spec and use a `,` instead for concatenation?

[RFC 2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2)